config/sample-local.yaml

```diff
@@ -8,13 +8,14 @@ app:
   site:
     logo:
     brand:
+  defaultChain: btc
```

Comment:
I think it default to `btc` even if that is not specifiied, right?

Reply:
The `default.yaml` file has the same settings. But repeating them in the local file makes it easier to see where/how you can change them.